### DISCUSSION TITLE
Migrated from Intel Mac Mini to M1 Mac Mini, and now can't get to transmission from outside container

### DISCUSSION THREAD

**Discussion by ssuess** [haugene/docker-transmission-openvpn#2830] [Q&A] [answered]
Hi, I can't figure out what is wrong after all the troubleshooting steps seem to indicate a well running container. Here is what I did:
- Installed the latest MacOS on the external boot drive (SSD) I formerly had attached to the Intel Mac, and it boots up the M1 perfectly
- Installed the Apple Silicon version of Docker (among other m1 versions of unrelated apps and Xcode, etc)
- enabled Rosetta so it would run
- ran my regular config that worked for years on my intel Mac Mini (Surfshark VPN btw)

Everything seems to start up fine, no complaints from the container
- boot logs look fine I think (minor warning about a deprecated option in OPVN)
- in console I properly get the VPN IP when doing curl ifconfig.me (so it seems to have indeed connected)
- in console curl localhost:9091 returns `<h1>301: Moved Permanently</h1>` as expected
- I can also ping outside addresses like google.com with no issue

In short everything seems fine, but I can't get to it anymore from the outside. Not on the mac host running the docker container, not from other machines on my local network. This all worked fine until I moved to the M1 Mac Mini from the Intel one.

Before you ask, of course I am doing this as before:
`-e LOCAL_NETWORK=192.168.10.0/24` and no, my local network has not changed, and the address assigned to the new host is the same as the old.

So I am stumped at this point. Anyone have any ideas? Thanks!

**Comment by ssuess** [comment]
After looking through a bunch of old posts, I found that the solution is STILL (even though docker is at version 4.29!) to revert to 4.22.1 which came out last August. It took me a while to find an archived download link, but here it is for anyone that needs it: 

https://desktop.docker.com/mac/main/arm64/118664/Docker.dmg

It seems like this will never be fixed by Docker. So perhaps we need some other workaround?

**Comment by ssuess** [comment]
maybe leave open for a bit for discussion?

**Comment by ssuess** [comment, marked as answer]
Finally there is a fix for this, detailed here: https://github.com/haugene/docker-transmission-openvpn/issues/2723
